README.md

```diff
@@ -1,11 +1,11 @@
-# Advent of code 2020 solutions
-
-⁣    🌟  
-    🎄  
-   🎄🎄  
-  🎄🎄🎄  
- 🎄🎄🎄🎄  
-🎄🎄🎄🎄🎄  
+# Advent of code 2020 solutions <!-- omit in toc --> <!-- do not remove comment + use Markdown All in One in VSCode -->
```

Comment:
When using the extension `Markdown All in One` in VSCode, using the comment `<!-- omit in toc -->` will remove this heading from the ToC